ambient.d.ts

```diff
@@ -468,12 +468,14 @@ declare module '@mojaloop/central-services-error-handling' {
 }
 declare module '@mojaloop/central-services-stream' {
   import { EventEmitter } from 'events';
-  export interface GenericMessage<T, U> {
+  export interface GenericMessage<EventType, EventAction> {
     value: {
       from: string,
       to: string,
       id: string,
       content: {
+        // Note: we don't know exactly what this will look like at this stage - as we have only inspected
```

Comment:
so maybe we should use Content as a generic type parameter, add an additional interface of Content for our case and later add more Content interfaces if needed?

Reply:
maybe also apply the same approach for metadata field?